Assets/LeapMotion/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -524,13 +524,60 @@ public class RuntimeGizmoDrawer {
     /// <summary>
     /// Draws a wire gizmo circle at the given position, with the given normal and radius.
     /// </summary>
-    public void DrawWireCirlce(Vector3 center, Vector3 direction, float radius) {
+    public void DrawWireCircle(Vector3 center, Vector3 direction, float radius) {
       PushMatrix();
       matrix = Matrix4x4.TRS(center, Quaternion.LookRotation(direction), new Vector3(1, 1, 0)) * matrix;
       DrawWireSphere(Vector3.zero, radius);
       PopMatrix();
     }
 
+    /// <summary>
+    /// Draws a wire gizmo capsule at the given position, with the given start and end points and radius.
+    /// </summary>
+    public void DrawWireCapsule(Vector3 start, Vector3 end, float radius) {
+      Vector3 up = (end - start).normalized * radius;
+      Vector3 forward = Vector3.Slerp(up, -up, 0.5F);
+      Vector3 right = Vector3.Cross(up, forward).normalized * radius;
+
+      float height = (start - end).magnitude;
+      float distToCap = Mathf.Max(0, (height * 0.5F) - radius);
+      Vector3 middle = (end + start) * 0.5F;
+
+      start = middle + ((start - middle).normalized * distToCap);
+      end = middle + ((end - middle).normalized * distToCap);
+
+      // Radial circles
+      DrawLineWireCircle(start, up, radius);
+      DrawLineWireCircle(end, -up, radius);
+
+      // Sides
+      DrawLine(start + right, end + right);
+      DrawLine(start - right, end - right);
+      DrawLine(start + forward, end + forward);
+      DrawLine(start - forward, end - forward);
+
+      // Endcaps
+      DrawWireArc(start, right, forward, radius, 26, 13);
+      DrawWireArc(start, forward, -right, radius, 26, 13);
+      DrawWireArc(end, right, -forward, radius, 26, 13);
+      DrawWireArc(end, forward, right, radius, 26, 13);
+    }
+
+    private void DrawLineWireCircle(Vector3 center, Vector3 normal, float radius) {
+      DrawWireArc(center, normal, Vector3.Slerp(normal, -normal, 0.5F), radius, 26, 26);
+    }
+
+    private void DrawWireArc(Vector3 center, Vector3 normal, Vector3 radialStartDirection, float radius, int numCircleSegments = 26, int numSegmentsToDraw = 26) {
+      normal = normal.normalized;
```

Comment:
Also, this is a super tiny thing, but I think that the default segment count should be divisible by 4 so that by default you can represent 25%, 50%, and 75% filled perfectly XD